onnxruntime/core/mlas/lib/intrinsics/avx2/min_max_elements.cpp

```diff
@@ -0,0 +1,107 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    min_max_elements.cpp
+
+Abstract:
+
+    This module implements the logic to find min and max elements with avx2 instructions.
```

Comment:
everything here appears to be AVX only: AVX has minps/maxps, so the kernel could be enabled for AVX, like MlasReduceMaximumF32KernelAvx.